src/packages/admin-ui-components/src/detail-panel/util.ts

```diff
@@ -0,0 +1,19 @@
+export const processFormValues = (formValues: any) => {
+	const result = Object.entries(formValues ?? []).reduce((acc, [key, value]: [string, any]) => {
```

Comment:
Maybe because it's a Friday afternoon 😄 But this code seems quite hard to follow. 

Now we have tests to cover its use could we rewrite it to make it easier to read?

I think maybe it's the reduce that could be replaced with a for-loop.

What do you think?